package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Serverless Framework, Apollo Server, and Contentful CMS",
   "main": "src/handlers/graphql.ts",
   "scripts": {
-    "dev": "sls offline start",
+  	"dev": "sls offline start",
```

Comment:
It looks like there's an extra tab here.

Reply:
Huh, so we were using spaces here, I indented the file correctly using tabs now